client/src/components/portfolio/CurrentPortfolio.js

```diff
@@ -24,15 +24,17 @@ class CurrentPortfolio extends Component {
     if (isEmpty(transactions)) {
       content = "Loading your current portfolio ...";
     } else {
-      const currentTransactions = this.state.transactions.map(transaction => (
```

Comment:
as this was depending on the state and the state would not be created, when the history push happens, caused the issue?

Reply:
yes